### ISSUE TITLE
Attaching a non existing file fails to load the webpage.

### ISSUE THREAD

**Issue by Apoorva-GA** [getgauge/taiko#69]
### Expected behavior
Should handle the error and shouldn't fail to load the page.

### Actual behavior
Fails to load the page

### Steps to reproduce
run script
```
const { openBrowser, goto, fileField, to, attach } = require('taiko');

(async () => {
    try {
        await openBrowser({headless: false});
        await goto('https://getgauge-examples.github.io/js-taiko/');
        await attach('file.txt', to(fileField('File')));
        await goto('https://getgauge-examples.github.io/js-taiko/');
        await focus('Username');
    } catch (e) {
        console.error(e);
    } finally {
        closeBrowser();
    }
})();
```

**Comment by Apoorva-GA** [comment]
Fix should be available in commit 79f3104c8c19487e22ac8457112374fb67d243eb

**Comment by sguptatw** [comment]
When the file does not exist, the script should fail by giving the error.

**Comment by Apoorva-GA** [comment]
Fix should be available in commit 59ab2e4
